docs/sdk/migrations/3.0.0/overview.md

```diff
@@ -239,13 +242,13 @@ All method interfaces in `@lit-protocol/encryption` have been updated.
 
 ## Minting Capacity Credits for Usage
 
-Currently Rate Limiting is enabled on `Habanero` and `Manzano`. In order to use these networks, you must reserve capacity on them by minting a `Capacity Credits NFT` on Chronicle - Lit's custom EVM rollup testnet. Capacity credits allow holders to reserve a configurable number of requests (measured in requests per second) over a fixed length of time (i.e. one week). For minting capacity credits, you can either use:
+Currently Rate Limiting is enabled on `Datil-prod` and `Datil-test`. In order to use these networks, you must reserve capacity on them by minting a `Capacity Credits NFT` on Chronicle - Lit's custom EVM rollup testnet. Capacity credits allow holders to reserve a configurable number of requests (measured in requests per second) over a fixed length of time (i.e. one week). For minting capacity credits, you can either use:
 1. The [Lit  Explorer](https://explorer.litprotocol.com/get-credits) or,
```

Comment:
The Lit Explorer no longer supports minting Capacity Credits

Reply:
Removed this, updated Chronicle -> Chronicle Yellowstone